clang/include/clang/Basic/Attr.td

```diff
@@ -1346,8 +1346,7 @@ def SYCLIntelKernelArgsRestrict : InheritableAttr {
 }
 
 def SYCLIntelNumSimdWorkItems : InheritableAttr {
-  let Spellings = [CXX11<"intelfpga","num_simd_work_items">,
-                   CXX11<"intel","num_simd_work_items">];
+  let Spellings = [CXX11<"intel","num_simd_work_items">];
```

Comment:
```suggestion
  let Spellings = [CXX11<"intel", "num_simd_work_items">];
```
Since we're touching the code anyway...  Also, the same formatting suggestion applies elsewhere in the lines you're touching.

Reply:
Thanks @AaronBallman for the reviews. I will make this change.